src/layout.rs

```diff
@@ -20,6 +20,10 @@ pub struct LayoutGlyph {
     pub rtl: bool,
     /// Cache key, see [CacheKey]
     pub cache_key: CacheKey,
+    // X offset in line
```

Comment:
If the docs could explain the difference between using this and x_int/y_int it would be good. Basically, if you want to cache with the CacheKey, you should use the x_bin/y_bin there.

Reply:
I tried to add some clarification in c46daa150106c070b1691d2bff8ffbd1a02f0e1f.